webapp/src/main/java/com/box/l10n/mojito/entity/PullRunTextUnitVariant.java

```diff
@@ -25,7 +26,7 @@
 @Table(
         name = "pull_run_text_unit_variant",
         indexes = {
-                @Index(name = "UK__PULL_RUN_TEXT_UNIT_VARIANT__EVA_ID__TM_TUV_ID", columnList = "pull_run_asset_id, tm_text_unit_variant_id", unique = true)
+                @Index(name = "UK__PULL_RUN_TEXT_UNIT_VARIANT__PRA_ID__TUV_ID__LOCALE_ID", columnList = "pull_run_asset_id, tm_text_unit_variant_id, locale_id", unique = true)
```

Comment:
Same comment as for v61 - locale_id would help queries most as the 2nd column in the index.